app/src/test/java/com/example/workoutapp/WorkoutUnitTest.kt

```diff
@@ -47,7 +43,6 @@ class WorkoutUnitTest {
         Dispatchers.setMain(mainThreadSurrogate)
         MockKAnnotations.init(this)
         gam = GroupsActivity()
```

Comment:
Rename this to a descriptive variable name.  Avoid acronyms and abbreviations, as they are not descriptive, and you won't remember what acronym/abbreviation you used later.